packages/cli/internal/pkg/aws/cdk/bootstrap.go

```diff
@@ -16,6 +18,28 @@ func (client Client) Bootstrap(appDir string, context []string, executionName st
 		"--tags", fmt.Sprintf("%s=%s", constants.AppTagKey, constants.AppTagValue),
 		"--profile", client.profile,
 	}
+
+	// Add AGC version and custom tags to the bootstrap stack.
+	agcVersionKey := "AGC_VERSION="
+	customTagsKey := "CUSTOM_TAGS="
```

Comment:
should these be constants? If `CUSTOM_TAGS` changes anywhere else, this will break

Reply:
Good point. It's a bit weird because we are going across TS and Go, but I pulled them out + other hardcoded constants being used here

Reply:
Yeah, nothing we can do about cross-language, that should be solved through a well documented API (or in our case, invocation parameters), but at least we can control it within the respective projects